### ISSUE TITLE
Package Control: Install Package not working

### ISSUE THREAD

**Issue by tarngerine** [wbond/package_control#1160]
looks like an issue with the cert? if i try to manually open it, keychain gives me an error
![image](https://cloud.githubusercontent.com/assets/1629901/18727107/33eeada8-8015-11e6-87ef-4f47257e0d85.png)

I'm on OS X 10.11.6, running Sublime Text 3. First time installing Package Control. Nothing happens when running "Install Package"

```
startup, version: 3114 osx x64 channel: stable
executable: /Applications/Sublime Text.app/Contents/MacOS/Sublime Text
working dir: /
packages path: /Users/tarng/Library/Application Support/Sublime Text 3/Packages
state path: /Users/tarng/Library/Application Support/Sublime Text 3/Local
zip path: /Applications/Sublime Text.app/Contents/MacOS/Packages
zip path: /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages
ignored_packages: []
pre session restore time: 0.206447
using gpu buffer for window
startup time: 0.269116
first paint time: 0.294629
reloading plugin Default.auto_indent_tag
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.quick_panel
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.show_scope_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin Vintage.vintage
reloading plugin Vintage.vintage_commands
reloading plugin Vintage.vintage_motions
reloading plugin 0_package_control_loader.00-package_control
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
plugins loaded
Package Control: Not running package cleanup since bootstrapping is not yet complete
reloading settings Packages/User/Package Control.sublime-settings
reloading /Users/tarng/Library/Application Support/Sublime Text 3/Packages/User/Package Control.sublime-settings
```

Probably the meat:
```
Exception in thread Thread-2:
Traceback (most recent call last):
  File "./python3.3/threading.py", line 901, in _bootstrap_inner
  File "package_control.commands.install_package_command in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 45, in run
  File "package_control.package_installer in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 58, in make_package_list
  File "package_control.package_manager in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 687, in list_available_packages
  File "package_control.package_manager in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 551, in _list_available
  File "package_control.package_manager in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 450, in list_repositories
  File "package_control.providers.channel_provider in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 201, in get_repositories
  File "package_control.providers.channel_provider in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 87, in fetch
  File "package_control.download_manager in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 340, in fetch
  File "package_control.downloaders.urllib_downloader in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 117, in download
  File "package_control.downloaders.urllib_downloader in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 317, in setup_opener
  File "package_control.ca_certs in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 31, in get_ca_bundle_path
  File "package_control.ca_certs in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 140, in get_system_ca_bundle_path
  File "package_control.deps.oscrypto.trust_list in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 115, in get_path
  File "package_control.deps.oscrypto._osx.trust_list in /Users/tarng/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 121, in extract_from_system
KeyError: b'\x8e\xdc\xee\x98\xf5x\x8d8\xb8\xd8\xad\x0e\x0ca7\xa6\xfb\xd1fm'
```

Thanks!

**Comment by jwalton512** [comment]
Exact same thing just started happening to me as well.

**Comment by tarngerine** [comment]
@jwalton512 thanks for the tip. i tried it but it didn't work. here's my setup:

![image](https://cloud.githubusercontent.com/assets/1629901/18729045/1b8fc5d6-801e-11e6-8de6-45ac0d17fac6.png)

did i put it in the wrong place

**Comment by tarngerine** [comment]
now it also duplicates every command here:
![image](https://cloud.githubusercontent.com/assets/1629901/18729085/45c9a326-801e-11e6-9308-a2835faea68d.png)

same error log

**Comment by tarngerine** [comment]
it seems to re add these files every time i launch sublime. im guessing thats expected? not sure why there's two of everything

![image](https://cloud.githubusercontent.com/assets/1629901/18729172/a2c0c050-801e-11e6-8092-d4e6e668b178.png)

**Comment by wbond** [comment]
@tarngerine You should clear all traces of Package Control and then clone the repo into `Packages/Package Control/`. Downloading the zip and extracting is causing you to put the files into a subdirectory. Since you didn't delete the `Installed Packages/Package Control.sublime-package` file, you now have duplicate resource files.

If you aren't comfortable with using git to clone, it would probably be best to wait until this fix is made into a release later tonight.

**Comment by wbond** [comment]
To add more info, the oscrypto-ca-bundle.crt is a PEM-formatted list of CA certs that are compatible with OpenSSL that have been exported from the OS X keychain. `Package Control.cache/` is where HTTP responses are cached, and `Package Control.sublime-settings` stores information about the current state of packages on your machine.

**Comment by tarngerine** [comment]
tonight? so fast :) i can wait

**Comment by philsinatra** [comment]
I tried cloning into `Packages/Package Control/.` and cleaning up the `.sublime-package` file in `Installed Packages`. I also tried both the `master` and `sox-trust-list` branches of the repo with no luck. Package Control basically has no command pallet options. I did just update to Sublime Text build 3124. I am still running macOS 10.11.6 El Capitan. I'll wait for your upcoming release to see if the issue is resolved.

![screen shot 2016-09-22 at 8 25 22 am](https://cloud.githubusercontent.com/assets/1465808/18748021/6caa7a44-809e-11e6-91ff-18fe5b7bdeb5.png)
